docs/kakao/io.github.kakaocup.kakao.picker.time/-time-picker-action/set-time.md

```diff
@@ -2,7 +2,7 @@
 
 # setTime
 
-`open fun setTime(hour: `[`Int`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-int/index.html)`, minutes: `[`Int`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-int/index.html)`): `[`ViewInteractionDelegate`](../../io.github.kakaocup.kakao.delegate/-view-interaction-delegate/index.md)
+`open fun setTime(hour: `[`Int`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-int/index.html)`, minutes: `[`Int`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-int/index.html)`): <ERROR CLASS>`
```

Comment:
It seems there is some typing issues coming with the update. Maybe regenerating dokka will fix that?

Reply:
@Unlimity probably dokka also should be uplifted

Reply:
Let's check it tommoz

Reply:
Guys, does anyone use dokka? Looks like a weird tool that brings nothing useful for users. What do you think?

Reply:
I don't know for sure, but I think there should be a way to look up API without pulling the library. And with these generated docs you can do it right on github. WDYT @Vacxe ?